lib/media/segment_utils.js

```diff
@@ -417,9 +417,13 @@ shaka.media.SegmentUtils = class {
       if (initData) {
         captionParser.init(initData);
       }
-      captionParser.parseFrom(data);
-      for (const stream of captionParser.getStreams()) {
-        closedCaptions.set(stream, stream);
+      try {
+        captionParser.parseFrom(data);
+        for (const stream of captionParser.getStreams()) {
+          closedCaptions.set(stream, stream);
+        }
+      } catch (e) {
+        // Ignore errors.
```

Comment:
Can we at least print error via `shaka.log.debug()`?

Reply:
Done!